WTWSMS/decisions/GR_Scythian.txt

```diff
@@ -203,7 +203,7 @@ decisions = {
 	}
 }
 
-targetted_decisions = {
+decisions = {
```

Comment:
> Wrong. You can consult a particular Shaman as a Tengri and have him see a vision, too, but that Decision is where it belongs.

That decision works differently though, it is only one of many options.

> The restrictions are fine: the fact that they are in a menu where people rarely, if ever, go to without notification makes them inaccessible simply because they cannot be found. So many people likely play this Mod not knowing that they can convert to local cultures and religions because someone was stupid enough to move those decisions off of where they were to that menu!

Huh? The diplomacy menu is constantly used, and you get notifications.